docs/datamap/lucene-datamap-guide.md

```diff
@@ -0,0 +1,180 @@
+# CarbonData Lucene DataMap
+  
+* [Quick Example](#quick-example)
+* [DataMap Management](#datamap-management)
+* [Lucene Datamap](#lucene-datamap-introduction)
+* [Loading Data](#loading-data)
+* [Querying Data](#querying-data)
+* [Data Management](#data-management-with-pre-aggregate-tables)
+
+## Quick example
+Download and unzip spark-2.2.0-bin-hadoop2.7.tgz, and export $SPARK_HOME
```

Comment:
Close all the sentence with a period (.). This is applicable for all the sentences in this topics.